src/thor/trace_route_action.cc

```diff
@@ -18,6 +18,7 @@ using namespace valhalla;
 using namespace valhalla::baldr;
 using namespace valhalla::sif;
 using namespace valhalla::thor;
+using std::vector;
```

Comment:
is this really needed? i dont see any where in this file where you declare a vector since you decided to move everything to member variables (which i also think is a needless "optimization")